clusterloader2/GETTING_STARTED.md

```diff
@@ -0,0 +1,239 @@
+  - namespaceRange:
+      min: 1
+      max: 1
+# Here we an specify how many replicas of object we want to create within namespace.
+    replicasPerNamespace: 1
+    tuningSet: Uniform1qps
+    objectBundle:
+    - basename: test-deployment
+# Path to file containing template for object to be created.
+      objectTemplatePath: "deployment.yaml"
+# Parameters to fill template.
+      templateFillMap:
+        Replicas: 1
+        Group: test-pod
+        CpuRequest: 1m
+        MemoryRequest: 10M
+# Once we started measurements and created deployment, now we can wait for this deployment to be running.
+- name: Wait for pods to be running
+  measurements:
+  - Identifier: WaitForControlledPodsRunning
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: gather
+# Now we can delete deployment.
+- name: Delete deployment
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    replicasPerNamespace: 0
+    tuningSet: Uniform1qps
+    objectBundle:
+    - basename: test-deployment
+      objectTemplatePath: "deployment.yaml"
+# And wait till it's actually deleted
+- name: Wait for pods to be deleted
+  measurements:
+  - Identifier: WaitForControlledPodsRunning
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: gather
+# At the end we want to collect pod startup latency.
+- name: Collecting pod startup latency
+  measurements:
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: gather
+```
+Now, we need to prepare template for our deployment to be created. All values in brackets will be filled from template parameters:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  labels:
+    group: {{.Group}}
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    matchLabels:
+      name: {{.Name}}
+  template:
+    metadata:
+      labels:
+        name: {{.Name}}
+        group: {{.Group}}
+    spec:
+      containers:
+      - image: k8s.gcr.io/pause:3.1
+        imagePullPolicy: IfNotPresent
+        name: {{.Name}}
+        resources:
+          requests:
+            cpu: {{.CpuRequest}}
+            memory: {{.MemoryRequest}}
+```
+## Execute test
+Before running test, make sure that kubeconfig current context points to kind cluster:
+```bash
+kubectl config current-context
+kind-test-cluster
+```
+To execute test, run:
+```bash
+cd clusterloader2/
+go run cmd/clusterloader.go --testconfig=config.yaml --nodes=1 --provider=kind --kubeconfig=${HOME}/.kube/config --masterip=127.0.0.1 --mastername=test-cluster --master-internal-ip=127.0.0.1
+```
+
+At the end of clusterloader output you should see pod startup latency and information that test succeeded:
+```
+PodStartupLatency: perc50: 709.983961ms, perc90: 709.983961ms, perc99: 709.983961ms
+PodStartupLatency: perc50: 2.709983961s, perc90: 2.709983961s, perc99: 2.709983961s
+PodStartupLatency: perc50: 2.709983961s, perc90: 2.709983961s, perc99: 2.709983961s; threshold 2m0s
```

Comment:
There is no way a newcomer would understand why we have so many lines here and what those mean.